src/main/java/betterquesting/client/QuestNotification.java

```diff
@@ -82,16 +82,12 @@ public void onDrawScreen(RenderGameOverlayEvent.Post event) {
 
             GL11.glEnable(GL11.GL_BLEND);
             GL11.glBlendFunc(GL11.GL_SRC_ALPHA, GL11.GL_ONE_MINUS_SRC_ALPHA);
-
-            if(alpha > 0.2F){
```

Comment:
Why are you removing this?
That's to make the text disappear at the same time as the icon... The text lingers for quite a while after the icon is gone.

Reply:
If anything the disappear value should be set lower to let them fade for longer. But it makes no sense to remove it and have the icon and text disappear at different times